bin/add-cmderplugin.ps1

```diff
@@ -0,0 +1,161 @@
+<#
```

Comment:
Since alias.bat was already there I thought it the best place to put it and 
the supporting utils.ps1.

Hopefully some people will try it out and we can start hosting plug-in json 
files along side the main repo.


On February 20, 2016 4:06:52 PM Jan Schulz <notifications@github.com> wrote:

>> @@ -0,0 +1,161 @@
>> +<#
>
> Ok, the readme doesn't say anything else than "it will be in path" and it 
> already has the alias.bat in it... Still seems untidy (like the stuff in 
> config until it got mostly reorganized), but ok... :-)
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cmderdev/cmder/pull/849/files#r53558456